data/src/main/kotlin/com/vultisig/wallet/data/api/EvmApi.kt

```diff
@@ -530,6 +533,21 @@ class EvmApiImp(
         return Numeric.toHexString(data?.copyOfRange(data.size - 20, data.size))
     }
 
+    private fun String.parseBalance(): BigInteger {
+        val fn = EthereumAbiFunction("balanceOf")
+        fn.addParamUInt256(ByteArray(32), true)
+        val dataHex = removePrefix("0x")
```

Comment:
please add removePrefix("0x") as extension in StringExtensions.kt

Question: wouldn't `toHexBytes()` works here? Please have a look at `String`, `ByteArray`, `BigInteger` extensions.